app/src/androidTest/java/org/mozilla/fenix/ui/TopSitesTest.kt

```diff
@@ -157,7 +157,7 @@ class TopSitesTest {
             expandMenu()
             verifyAddToTopSitesButton()
         }.addToFirefoxHome {
-            verifySnackBarText("Added to top sites!")
+            verifySnackBarText("Added to shorcuts!")
```

Comment:
should we make this a string resource reference, or does that still pose the same fragility?